channels/views_test.py

```diff
@@ -4,12 +4,23 @@
 from rest_framework import status
 
 from open_discussions.factories import UserFactory
+from profiles.factories import ProfileFactory
+from channels.serializers import default_profile_image
 
 
 # pylint: disable=redefined-outer-name, unused-argument
 pytestmark = pytest.mark.django_db
 
 
+def create_profile_and_login(client):
+    """little helper function"""
+    profile = ProfileFactory.create()
```

Comment:
You can set attributes via the dunder syntax with factories, like `ProfileFactory.create(user__username='george')`